js/whosehouse.js

```diff
@@ -57,6 +59,16 @@ function displayWhoseHouseWinner() {
   baselineRotation = Math.floor(finalRotation / 360) * 360;
   console.log('houses: ' + wh.houses.join());
   console.log('winner: ' + winnerHouse);
+
+  // Bold the winner when done rotating
+  $('#wheel').bind(transition_end_events,
+      (function(winnerHouse) {
+        return function() {
+          $('#wheel-' + winnerHouse).css('font-weight', 'Bold');
+          $(this).unbind(transition_end_events)
```

Comment:
You can move the `bind` call (line 64) outside of the `displayWhoseHouseWinner`, so that the event listener is attached only once, on page load. Then you can remove the call to `unbind`.

Reply:
how do you make it get the winnerHouse?

Reply:
Oh, you are right. Sorry, I was hasty in reading the code. (Which is actually not so different from some code reviews I give at work sometimes >_< ) Disregard this comment.